spec/codecs/json_spec.rb

```diff
@@ -147,6 +150,24 @@
           end
         end
       end
+            expect( event.get("[root][baz]")['1'] ).to be true
```

Comment:
something is funky with the diff here -- the result is `expect` outside of examples.